app/Kconfig

```diff
@@ -78,6 +78,18 @@ config ZMK_BLE_CLEAR_BONDS_ON_START
 	bool "Configuration that clears all bond information from the keyboard on startup."
 	default n
 
+config BT_PERIPHERAL_PREF_MIN_INT
+	default 6
```

Comment:
Apple’s guide states:

> The connection parameter request may be rejected if it does not meet the following guidelines:
> - Peripheral Latency of up to 30 connection intervals.
> - Supervision Timeout from 2 seconds to 6 seconds.
> - Interval Min of at least 15 ms.
> - Interval Min is a multiple of 15 ms.
> - One of the following:
>   - Interval Max at least 15 ms greater than Interval Min.
>   - Interval Max and Interval Min both set to 15 ms.
> - Interval Max * (Peripheral Latency + 1) of 2 seconds or less.
> - Supervision Timeout greater than Interval Max * (Peripheral Latency + 1) * 3.

So I think this means that, to match that guidance, this parameter should be 12 (in units of 1.25 ms).